frontend/src/styles/utils.scss

```
@@ -61,7 +61,8 @@ $color2_light_green: #DBFA70;
 $color3_purple: #CC71FA;
 $color4_orange: #FAA870;
 
-$main-app-margin-size: 10px;
+$main-app-margin-size: 32px;
```

Stylelint (17.14.1)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)